rajnikantBot.log

```diff
@@ -542,3 +542,2385 @@ Please check https://pkg.go.dev/github.com/gin-gonic/gin#readme-don-t-trust-all-
 [GIN] 2022/01/15 - 21:54:17 | 200 |  433.379956ms | 2405:201:a407:91cb:3535:bf33:7158:c569 | POST     "/rajnikantBot/create_2?userName=NolanBot"
 2022/01/15 21:54:54 61e2f55625cdd587f669f5ee
 [GIN] 2022/01/15 - 21:54:54 | 200 |   43.545536ms | 2405:201:a407:91cb:3535:bf33:7158:c569 | POST     "/rajnikantBot/create_3?bot_id=61e2f53025cdd587f669f5ed&pic_url=https://rukminim1.flixcart.com/image/416/416/kdukgi80-0/poster/2/z/k/medium-inception-minimal-black-framed-medium-inception-movie-original-imafunn4za5m4kyf.jpeg?q=70"
+[GIN-debug] [WARNING] Now Gin requires Go 1.13+.
```

Comment:
Lets remove the log file from here @shashank404error . The thing is .log is already added in .gitignore file but since this was already added by me before this log was in gitignore it is getting tracked by git.(AFAIK).